### PR TITLE
Update EIP-4361: Move to Review

### PR DESCRIPTION
**Pull request by obstropolos**
Update EIP-4361 back to `Review` from `Stagnant`.

### CODE REVIEW

**Review by SamWilsn — COMMENTED** [review]
You'll probably run into stability problems if you depend/link to so many EIPs, meaning you won't be able to advance your EIP's status past theirs. For example EIP-634 is Stagnant, so if you link to it, you'll be stuck in Draft. You can either remove these links or help those authors push their EIPs forward.

File: EIPS/eip-4361.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Sign-In with Ethereum
 description: Off-chain authentication for Ethereum accounts to establish sessions.
 author: Wayne Chang (@wyc), Gregory Rocco (@obstropolos), Brantly Millegan (@brantlymillegan), Nick Johnson (@Arachnid)
 discussions-to: https://ethereum-magicians.org/t/eip-4361-sign-in-with-ethereum/7263
-status: Stagnant
+status: Review
 type: Standards Track
 category: ERC
 created: 2021-10-11
```
[the remark this answers is not in the document]
Comment (obstropolos) [reply EIPS/eip-4361.md:10]: > You'll probably run into stability problems if you depend/link to so many EIPs, meaning you won't be able to advance your EIP's status past theirs. For example EIP-634 is Stagnant, so if you link to it, you'll be stuck in Draft. You can either remove these links or help those authors push their EIPs forward.

Right now, I thought we only rely on 55, 137, 155, 191, 1271, and 1328, with 1328 being the only one that isn't `Final` and also requires no external EIPs (working with @pedrouid here). 

Are there other situations to look out for?

[the remark this answers is not in the document]
Comment (obstropolos) [reply EIPS/eip-4361.md:10]: Ah I'm seeing this now. So this guideline in particular is a recommendation but not mandatory - the usage of `MAY` in `The relying party or wallet MAY` for EIP-634. 

Is there a way to note that so it isn't caught as a critical requirement?

[the remark this answers is not in the document]
Comment (Pandapip1) [reply EIPS/eip-4361.md:10]: Just remove it from the requires list!

**Comment by obstropolos** [comment]
Alrighty I think I finally fixed all the issues! Praise be the linter and EIP Walidator.

@Pandapip1 / @SamWilsn

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**